cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -349,7 +350,7 @@ func (c *Client) WaitForClusterAvailable(retries int, retryInterval time.Duratio
 			time.Sleep(retryInterval)
 		}
 		klog.V(2).Infof("[etcd] attempting to see if all cluster endpoints (%s) are available %d/%d", c.Endpoints, i+1, retries)
-		resp, err := c.ClusterAvailable()
+		_, err := c.getClusterStatus(true)
```

Comment:
@tedyu
the removal of ClusterAvailable() seems good to me, but i don't see the benefits of the true/false flag to create or not create a map with responses. IMO it should be always created and returned, we are not chasing CPU cycles and nanoseconds here.